can-stache-bindings.js

```diff
@@ -1283,6 +1265,29 @@ var makeDataBinding = function(node, el, bindingData) {
 	// This completes the binding.  We can't call it right away because
 	// the `viewModel` might not have been created yet.
 	var completeBinding = function() {
+
+		//!steal-remove-start
+		if(
+			bindingInfo.child === "viewModel" &&
+			!isBindingsAttribute(bindingInfo.bindingAttributeName)
+		) {
+			// Check to see if the viewModel has the attribute name as a defined property
+			var viewModel = bindingData.getViewModel();
```

Comment:
Can you use the `childObservable` here instead of getting it from the `bindingData`? It should always be the viewModel. It probably doesn't make a big difference, so however you want to do it is fine.

Reply:
`childObservable` is an Observation; how could I use it instead?